todoapp/projects/models.py

```diff
@@ -1,6 +1,11 @@
-
 from django.db import models
 
+from todoapp.settings import AUTH_USER_MODEL
```

Comment:
how is these different, what should be used?
```suggestion
from django.conf import settings
```